### PR TITLE
Add documentation for the new arithmetic Processor

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Add documentation for the new arithmetic processor

### Motivation
Customer wanted to be able to perform arithmetic operations on their log attributes. This processor was added and it therefore requires documentation.

### Preview link

* https://docs-staging.datadoghq.com/nils/arithmetic-processor/logs/processing/processors/#arithmetic-processor

### CODE REVIEW

File: content/logs/processing/processors.md [hunk 1]
```diff
@@ -138,6 +138,25 @@ The message is a key attribute in Datadog. It is displayed in the message column
...
+Use the Arithmetic Processor to add a new attribute (without spaces or special characters in the new attribute name) to a log with the result of the provided formula.
+This can be very useful to remap different time attributes with different units into a single attribute. Or to compute operations on attributes within the same log.
+
+The formula can use parenthesis and the basic arithmetic operators: `-`, `+`, `*`, `/`. 
+
+Example:
+
+{{< img src="logs/processing/processors/arithmetic_processor.png" alt="Arithmetic Processor" responsive="true" style="width:80%;">}}
+
+By default the calculation is skipped if an attribute is missing but it is possible to replace the missing attribute value by 0 to ensure that the calculation is done.
```
Comment (cswatt) [inline content/logs/processing/processors.md]: We should also explain how to replace the missing attribute value.

Comment (NBParis) [reply content/logs/processing/processors.md]: @cswatt not sure I understand your point. What do you mean exactly?

Comment (NBParis) [reply content/logs/processing/processors.md]: I thought the screenshot was self explanatory enough: https://cl.ly/f24c15c86c1c

But let me know if we should highlight the checkbox more.

Comment (cswatt) [reply content/logs/processing/processors.md]: Wow, it was very early and I did not see the checkbox. However I think I'd still like this to be more explicit in the text. I'll add to this branch.

**Pull request merged**